src/zap/zapimport.cpp

```diff
@@ -1674,7 +1674,7 @@ class ZapDynamicHelperCell : public ZapImport
 
     CORCOMPILE_FIXUP_BLOB_KIND GetKind()
     {
-        int kind = (int)GetHandle();
+        int kind = (int)(SIZE_T)GetHandle();
```

Comment:
I think that `kind` can just be typed to `SIZE_T`, remove the `int`.

Reply:
Much cleaner, thanks!